GenderPayGap.Core/App_Data/CustomErrorMessages.config

```diff
@@ -36,6 +36,7 @@
     <CustomErrorMessage code="3015" title="There's a problem with your PIN" description="You've entered an incorrect PIN. Please try again"/>
     <CustomErrorMessage code="3017" title="This employer was previously registered by {approvedUser} on {approvedDate} at {approvedAddress}" />
     <CustomErrorMessage code="3018" title="{requestCount} other pending registration request(s) exist for this employer" />
+    <CustomErrorMessage code="3023" title="You can't remove this registration yet." description="Please try again in {remainingTime}" actionUrl="/manage-organisations"/>

```

Comment:
`manage-organisations` is not a valid URL anymore. should be replaced with `/account/organisations`